session/common.go

```diff
@@ -308,6 +308,16 @@ func (x *commonData) SetAuthKey(key neofscrypto.PublicKey) {
 	x.authKey = x.authKey[:key.Encode(x.authKey)]
 }
 
+// SetIssuer allows to set issuer before sign call.
+func (x *commonData) SetIssuer(signer neofscrypto.Signer) error {
```

Comment:
Why not just `SetIssuer(id user.ID)`?

Reply:
Just tried to save method signature, updated